### PR TITLE
Added front end multilingual setup

### PR DESCRIPTION
**Pull request by jmayntzhusen**

### CODE REVIEW

**Comment by jmayntzhusen** [comment]
#655 Updated front end docs for multilingual sites

File: Tutorials/Multilanguage-Setup/index.md [hunk 1]
```diff
@@ -1,8 +1,55 @@
-#Multilanguage sites#
-_This section features some tips on how to translate labels for you document types like tabs, name and description. For more information on how to create a multilanguage frontend website please read this article [1-1 Multilingual Websites in Umbraco
-](http://24days.in/umbraco/2012/multilingual-1-1/)_
+#Multilanguage sites
 
-##Multilanguage backoffice##
+##Multilingual frontend
+_Note: This will be shown on Umbraco 7.8.1, never and older versions may differ. Also this assumes you already have a site with content in one language ready to work with._
```
Comment (dampee) [inline Tutorials/Multilanguage-Setup/index.md]: Should be "newer" instead of "never".

**Comment by jmayntzhusen** [comment]
@dampee I fixed the typo, thanks for the catch :)

**Comment by jmayntzhusen** [comment]
Hey @dampee 

It should work now, thanks for reviewing it :)
- Jesper

**Comment by dampee** [comment]
Awesome. Thanks a lot for your hard work. I am sure a lot of people will we better of now  ;-)

H5YR!

**Pull request merged**